companion/src/modeledit/setup.cpp

```diff
@@ -238,9 +238,9 @@ ModulePanel::ModulePanel(QWidget * parent, ModelData & model, ModuleData & modul
   lock = false;
 
   // a change will not register if only one item was added
-  if (ui->protocol->count() == 1)
+  if (ui->protocol->count() == 1) {
```

Comment:
I'm with Andre.  :)   Especially:
> But NOT the single-line construction.

I will take the "extra" brackets over that any day.

Here's where we left it last time... :)  
https://github.com/opentx/opentx/pull/4141#discussion_r92862868